lib/util/Components.js

```diff
@@ -79,7 +79,7 @@ Components.prototype.set = function(node, props) {
  *
  * @returns {Object} Components list
  */
-Components.prototype.list = function(filter) {
+Components.prototype.list = function() {
```

Comment:
`filter`is used in the function. Are you sure to be OK if we remove?

Reply:
In this case, the `filter` from the function parameter is not used. The used `filter` is the method from the `Array` instance